experiments.json

```diff
@@ -60,5 +60,19 @@
       "min": "75",
       "max": "100"
     }
-  }
+  },
+  "promote-add-to-homescreen": {
+    "match": {
+      "appId": "^org.mozilla.fennec$"
+    },
+    "buckets": {
+      "min": 50,
+      "max": 100
+    },
+    "values": {
+      "minimumTotalVisits": 5,
+      "lastVisitMinimumAge": 30000,
```

Comment:
Maybe you should rename this `lastVisitMinimumAgeMs` (assuming it is indeed ms). Or maybe choose a short unit of time so it's more intuitive to read this.